### PR TITLE
Add loading of env specific module configs from main config dir

### PR DESCRIPTION
**Pull request by iturgeon**
To keep the environment specific code for modules centralized, I added the option to place them in the main app config directory.  This will keep all the environment specific configs near each other, instead of buried throughout the file system.

This change allows configs for modules to be located in:
/fuel/app/module/modname/config/cfg.php
/fuel/app/module/modname/config/environment/cfg.php
..and now..
/fuel/app/config/environment/module/modname/cfg.php

Signed-off-by: Ian Turgeon iturgeon@gmail.com

### CODE REVIEW

File: classes/config/file.php [hunk 1]
```diff
@@ -126,8 +126,15 @@ protected function find_file()
 			return $paths;
 		}
 
+		// look for environment module configs in app/modules/mod/config/env/...
```
Comment (iturgeon) [inline classes/config/file.php:129]: This comment is incorrect, its not just for modules... will fix

**Comment by WanWizard** [comment]
As this is a functional change, it needs to be a pull request on 1.3/develop, not on 1.2/develop.

**Comment by WanWizard** [comment]
Also, your you send a pull request on fuel/docs, 1.3/develop, to document this new feature?

**Comment by WanWizard** [comment]
Closed as incorrect framework usage.

**Pull request closed without merging**